example/notebooks/pneumonia.ipynb

```diff
@@ -0,0 +1,1083 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Explanation analysis using pneumonia XRay pictures with FoXAI\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Double check if you need to reinstall linux libaries related to OpenCV"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!apt-get update && apt-get install ffmpeg libsm6 libxext6  -y"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Installing most needed libaries "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install scikit-learn foxai tqdm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "e_eezmSZJJe7"
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "import torch\n",
+    "import torchvision\n",
+    "import torch.nn as nn\n",
+    "import torch.optim as optim\n",
+    "from torchvision import datasets, models, transforms\n",
+    "from torch.utils.data.sampler import SubsetRandomSampler\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import time\n",
+    "import copy\n",
+    "from random import shuffle\n",
+    "\n",
+    "import tqdm.notebook as tqdm\n",
+    "\n",
+    "import sklearn\n",
+    "from sklearn.metrics import accuracy_score, cohen_kappa_score\n",
+    "from sklearn.metrics import classification_report\n",
+    "from PIL import Image\n",
+    "import cv2\n",
+    "\n",
+    "import os\n",
+    "import shutil"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Train Dataset Exploration and Creation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For the purpose of this notebook we create dataset from 3 sources: \n",
+    "1. [Open dataset of Covid-19 cases with chest X-ray](https://github.com/ieee8023/covid-chestxray-dataset)\n",
+    "2. [Paul Mooney's pneumonia dataset from ](https://www.kaggle.com/datasets/paultimothymooney/chest-xray-pneumonia)\n",
+    "3. [Kaggle's Covid-19 radiography database](https://www.kaggle.com/datasets/tawsifurrahman/covid19-radiography-database)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "_kg_hide-input": true,
+    "_kg_hide-output": true,
+    "id": "ZghMYdMHMGuj"
+   },
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv('./covid-chestxray-dataset/metadata.csv')\n",
```

Comment:
Missing file. You should explain how to download/create this file as it is not present in the FoXAI repository. You may add a sentence that user have to manually download the dataset from Kaggle and clone the GitHub repository.

Maybe at the cell above add something like this:
`!git clone https://github.com/ieee8023/covid-chestxray-dataset`

Reply:
fixed